yt/units/tests/test_ytarray.py

```diff
@@ -828,7 +828,10 @@ def unary_ufunc_comparison(ufunc, a):
 
 
 def binary_ufunc_comparison(ufunc, a, b):
-    out = a.copy()
+    if ufunc in [np.divmod]:
```

Comment:
```suggestion
    if ufunc is np.divmod:
```
No strong opinion about this though. I get that your syntax scales better in case we need to add other special cases in the future.